### PR TITLE
Feature fixed navbar

### PR DESCRIPTION
**Pull request by purisaurabh**
This PR contains the navbar part of the profile-builder application

### CODE REVIEW

File: src/components/Navbar/navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        title="Confirm Logout"
+        open={isModalVisible}
+        onOk={handleOk}
+        onCancel={handleCancel}
+        okText="Yes"
+        cancelText="No"
+      >
+        <p>Are you sure you want to logout?</p>
+      </Modal>
+      {isLogout && <Logout />}
```
Comment (Payal13) [inline src/components/Navbar/navbar.jsx]: on logout you should directly call a function which will clear the tokens. no need of any component and isLogout state.

File: src/components/Logout/logout.js
Comment (Payal13) [inline src/components/Logout/logout.js:1]: No need of this component

File: src/routes/router.js [hunk 2]
```diff
@@ -25,6 +26,10 @@ const Router = () => {
           path="/profile-builder"
           element={<ProtectedRoutes Component={Editor} />}
         />
+        <Route
```
Comment (Payal13) [inline src/routes/router.js]: No need of any separate route for logout as its just a function.
after logout it should get redirected to login

File: src/components/Resume/Resume.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ import React, { forwardRef, useEffect, useRef, useState } from "react";
 import { Calendar } from "react-feather";
 import joshImage from "../../assets/Josh-Logo-White-bg.svg";
 import { genderOptions, getMonthString } from "../../Constants";
+import jsonData from "./jsonData.json";
```
Comment (Payal13) [inline src/components/Resume/Resume.js]: remove unnecessary code. this PR should only have navbar and layout related changes.

File: src/components/Builder/Index.jsx [hunk 4]
```diff
@@ -82,88 +83,88 @@ export const Editor = () => {
...
+          lg={{ span: 12 }}
+          className={styles["hide-scrollbar"]}
           style={{
-            display: "flex",
-            justifyContent: "center",
-            marginTop: "10px"
+            minheight: "98vh",
+            maxHeight: "98vh",
+            overflow: "auto",
+            padding: "2%",
```
Comment (Payal13) [inline src/components/Builder/Index.jsx]: lets use rem instaed of percent here as well

File: src/components/Login/index.js [hunk 5]
```diff
@@ -21,7 +21,7 @@ const Login = () => {
         if (token) {
           dispatch(loginAction({ token }));
           window.localStorage.setItem("token", token);
```
Comment (Payal13) [inline src/components/Login/index.js:24]: it should get stored in reducer as well from here. check from where we are currently stoeing it?

**Review by Payal13 — CHANGES_REQUESTED** [review]

File: src/components/Builder/Index.jsx (hunk 4 above)
Comment (purisaurabh) [reply src/components/Builder/Index.jsx]: kk

File: src/components/Resume/Resume.js (hunk 3 above)
Comment (purisaurabh) [reply src/components/Resume/Resume.js]: done

File: src/components/Login/index.js (hunk 5 above)
Comment (purisaurabh) [reply src/components/Login/index.js:24]: already implemented

File: src/routes/router.js (hunk 2 above)
Comment (purisaurabh) [reply src/routes/router.js]: kk done

File: src/components/Logout/logout.js
Comment (purisaurabh) [reply src/components/Logout/logout.js:1]: kk

File: src/components/Navbar/navbar.jsx (hunk 1 above)
Comment (purisaurabh) [reply src/components/Navbar/navbar.jsx]: done

**Review by Payal13 — APPROVED** [review]

**Pull request merged**